vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -437,6 +488,11 @@ export default class SbcHeader extends Mixins(NavigationMixin) {
       // checking for account status
       await this.checkAccountStatus()
     }
+
+    await this.syncNotifications()
+    await this.fetchNotificationCount()
+    await this.fetchNotificationUnreadPriorityCount()
+    await this.fetchNotificationUnreadCount()
```

Comment:
I'm wondering if these could be done in the background so they don't slow down the loading of the sbc header...?

Reply:
How?

Reply:
One way is to not await them -- assuming they are not dependents on each other -- and if you do this, add a comment stating that you are intentionally not awaiting an async method (and why you're doing it).

Reply:
All these function tight together have to wait. But it should not cause performance issue. I will put the comment in it.

Reply:
OK, I'm glad you thought about it.